backend/src/db/helpers/migration-helper.ts

```diff
@@ -0,0 +1,34 @@
+import { type Knex } from 'knex';
+
+import { CommonTableColumns } from '#libs/enums/enums.js';
+
+function createTableWithCommonColumns(
```

Comment:
As @Xxandder mentioned, we need to create an independent migrations. So your idea is good, bit not suitable for migrations